Pal/lib/assert.h

```diff
@@ -27,9 +27,8 @@ void __abort(void);
             ? ({                                                        \
                     warn("assert failed " __FILE__ ":%d " #test " (value:%x)\n", \
                          __LINE__, _val);                               \
-                    __abort(); })                                       \
```

Comment:
I'm not sure about this change.  Why the while(1)?  I would think you really want to terminate a program that blows an assertion.  Or is this just for debugging?

Reply:
Debugging leftover. This PR was created by Chia-Che straight from my development branch, so please be very careful, there may be more of such things hiding here ;)